src/devtools/store.js

```diff
@@ -572,7 +573,52 @@ export default class Store extends EventEmitter {
             weightDelta = 1;
           }
           break;
-        case TREE_OPERATION_REMOVE:
+        case TREE_OPERATION_CLEAR_CHILDREN: {
+          id = ((operations[i + 1]: any): number);
+
+          if (!this._idToElement.has(id)) {
+            throw new Error(
+              'Store does not contain fiber ' +
+                id +
+                '. This is a bug in React DevTools.'
+            );
+          }
+
+          i = i + 2;
+
+          let justRemovedIDs = [];
+          const recursivelyRemove = id => {
+            justRemovedIDs.push(id);
+            const el = this._idToElement.get(id);
+            if (!el) {
+              throw new Error(
+                'Store does not contain fiber ' +
+                  id +
+                  '. This is a bug in React DevTools.'
+              );
+            }
+            this._idToElement.delete(id);
+            el.children.forEach(recursivelyRemove);
+          };
+
+          // Track removed items so search results can be updated
+          const oldRemovedElementIDs = removedElementIDs;
+          removedElementIDs = new Uint32Array(
+            removedElementIDs.length + justRemovedIDs.length
+          );
+          removedElementIDs.set(oldRemovedElementIDs);
+          let startIndex = oldRemovedElementIDs.length;
+          for (let j = 0; j < justRemovedIDs.length; j++) {
+            removedElementIDs[startIndex + j] = oldRemovedElementIDs[j];
+          }
+
+          parentElement = ((this._idToElement.get(id): any): Element);
+          parentElement.children.forEach(recursivelyRemove);
+          parentElement.children = [];
+          weightDelta = -parentElement.weight + 1;
+          break;
```

Comment:
Kind of a bummer that this requires so much overlap with the `TREE_OPERATION_REMOVE` operation.